### DISCUSSION TITLE
使用多工作区 i18n 插件问题记录

### DISCUSSION THREAD

**Discussion by mojiefong** [vbenjs/vue-vben-admin#4459] [General]
使用 v5新版 后，打开根目录的 `vben-admin.code-workspace` 文件启用多工作区模式后 `i18n Ally` 插件不工作的问题

<img width="1149" alt="image" src="https://github.com/user-attachments/assets/cac37fa9-c1ec-4879-b6b2-9f7795f5c372">

代码中并不会实时看到文案，日志也没有啥问题

准备给 `i18n Ally` 插件提个 issue，发现 `i18n Ally` 已经不活跃了，差不多快一年没更新了😂😂😂

所以只是在此记录一下

**Comment by mojiefong** [comment]
不过也能临时处理一下

把根目录 `.vscode/settings.json` 文件里的 `i18n` 相关配置都复制到 `vben-admin.code-workspace`

```jsonc
// vben-admin.code-workspace

"folders": [...],

// 配置工作区的 i18n
"settings": {
  "i18n-ally.localesPaths": [
    "packages/locales/src/langs",
    "playground/src/langs",
    "apps/*/src/locales/langs",
  ],
  "i18n-ally.enabledParsers": ["json", "ts", "js", "yaml"],
  "i18n-ally.sourceLanguage": "en",
  "i18n-ally.displayLanguage": "zh-CN",
  "i18n-ally.enabledFrameworks": ["vue"],
}
```

重启编辑器后发现有点感觉了，但是依旧无法加载

<img width="1134" alt="image" src="https://github.com/user-attachments/assets/0e14f50d-9cc6-4379-8385-c9da5ead4ad0">

然后在子项目中也需要创建 `.vscode/settings.json` ，以 `@vben/web-naive` 为例

``` json
{
  "i18n-ally.localesPaths": [
    "../../packages/locales/src/langs",
    "src/locales/langs"
  ]
}

```

<img width="1485" alt="image" src="https://github.com/user-attachments/assets/6084e86c-033f-46ae-8873-61b5dd40f915">

看日志已经加载成功了，那么再重启一下编辑器就OK了

这种方式比较繁琐，每个子项目都需要加一次，也就是哪里需要加哪里

只是单纯做个记录，如有需要的朋友可以自取，或者有更好的方式给我说下

另外就是 `i18n Ally` 没有维护了，不然应该不至于这样

**Comment by cc123hh** [comment]
改成绝对的地址好像也可以，但是项目位置可能会发生变化，虽然只用修改一次也不会经常变化位置
